example.env

```diff
@@ -0,0 +1 @@
+REACT_APP_APIKEY= 707b632968c44413dde9c0c67fbf2f25
```

Comment:
The API Key shouldnt be visible. Please add the env file to gitignore.

Reply:
I think Omkar accidentally added his API key in the `example.env` file. 
It's better to delete both `example.env` and `package-lock.json` and add the API key to the `.env` file before merging.